themes/getting-started/theme-organization.md

```diff
@@ -96,12 +99,15 @@ The folders are used this way:
 **/config/**
 : Contains configuration file. By default, it only has the theme.yml file.
 
+**/dependencies/modules/**
+: Contains theme-specific modules. *This folder isn't present in the Classic Theme because it has no theme-specific modules.*
```

Comment:
@SherloxTV we need a link ref here, right? I couldn't change anything in the second PR, let's focus on finishing this one and then we can backport it to v8 👍